.idea/runConfigurations/CI_Tests.xml

```diff
@@ -0,0 +1,30 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="CI Tests" type="TestNG" factoryName="TestNG">
+    <extension name="coverage" enabled="false" merge="false" sample_coverage="true" runner="idea" />
+    <module name="" />
+    <option name="ALTERNATIVE_JRE_PATH_ENABLED" value="true" />
+    <option name="ALTERNATIVE_JRE_PATH" value="1.7" />
+    <option name="SUITE_NAME" value="$PROJECT_DIR$/src/test/testng-ci.xml" />
+    <option name="PACKAGE_NAME" value="" />
+    <option name="MAIN_CLASS_NAME" value="" />
+    <option name="METHOD_NAME" value="" />
+    <option name="GROUP_NAME" value="" />
+    <option name="TEST_OBJECT" value="SUITE" />
+    <option name="VM_PARAMETERS" value="-ea -Duser.timezone=UTC" />
```

Comment:
Add `-Xss2m` parameter as well, like how it was done in the Eclipse variant.
(although, admittedly, I'm not sure why 😛)